sklearn/utils/estimator_checks.py

```diff
@@ -389,6 +391,8 @@ def _construct_instance(Estimator):
                 estimator = Estimator(LinearRegression())
             elif issubclass(Estimator, RegressorMixin):
                 estimator = Estimator(Ridge())
+            elif issubclass(Estimator, SelectFromModel):
+                estimator = Estimator(SGDRegressor(random_state=0))
```

Comment:
Let's include a comment so future contributors know why we need this:

```suggestion
            elif issubclass(Estimator, SelectFromModel):
                # Increases coverage because SGDRegressor has partial_fit
                estimator = Estimator(SGDRegressor(random_state=0))
```